src/pages/profile/Profile.tsx

```diff
@@ -0,0 +1,56 @@
+import { FC, useEffect, useState } from "react";
+import profilepic from "../../assets/profile.jpg";
+import { Button } from "@/common/button";
+import { Error } from "@/common/error";
+import { useParams } from "react-router-dom";
+
+export const Profile : FC = ()=>{
+    const [posts, setPosts] = useState([]);
+    const { id } = useParams()
+
+    useEffect(()=>{
+        console.log(id); //Get user by id
+    })
+
+    return(
+        <div>
```

Comment:
ensure This div fills the parent for better scrolling when It overflows

first, check, if it doesn't add `h-screen` or `h-full` (verify which one works better)